desktop/ui/src/main/java/org/datacleaner/panels/AddDatastorePanel.java

```diff
@@ -240,17 +212,17 @@ public void windowClosed(WindowEvent e) {
             public void actionPerformed(ActionEvent event) {
                 Injector injectorWithNullDatastore = _dcModule.createInjectorBuilder().with(datastoreClass, null)
                         .createInjector();
-                final AbstractDatastoreDialog<D> dialog = injectorWithNullDatastore.getInstance(dialogClass);
+                final AbstractDialog dialog = injectorWithNullDatastore.getInstance(dialogClass);
 
                 dialog.setVisible(true);
-                dialog.addWindowListener(new WindowAdapter() {
-                    @Override
-                    public void windowClosed(WindowEvent e) {
-                        if (dialog.getSavedDatastore() != null) {
-                            _datastoreSelectedListener.datastoreSelected(dialog.getSavedDatastore());
-                        }
-                    }
-                });
+                // dialog.addWindowListener(new WindowAdapter() {
+                // @Override
+                // public void windowClosed(WindowEvent e) {
+                // if (dialog.getSavedDatastore() != null) {
+                // _datastoreSelectedListener.datastoreSelected(dialog.getSavedDatastore());
+                // }
+                // }
+                // });
```

Comment:
Oh, I forgot this. Will double check that it is not necessary anymore and delete.